hack/quicklab/README.md

```diff
@@ -0,0 +1,46 @@
+# Configuring NFS storage provisioner on Quicklab clusters 
```

Comment:
It would be good to mention here that this install an operator, and link to the upstream project.

Reply:
added references and an explanation.